content/engine/_index.md

```diff
@@ -66,12 +66,7 @@ such as images, containers, networks, and volumes.
 For more details, see
 [Docker Architecture](../get-started/overview.md#docker-architecture).
 
-{{< grid >}}
-
-## Licensing
+Use of Docker Engine or Docker CLI through Docker Desktop is subject to the
```

Comment:
Should we update here as well to match below?

Reply:
I think it's fine to keep the current wording on the overview page